### PR TITLE
Add key_name attribute to server class

### PR DESCRIPTION
**Pull request by gabecase**

### CODE REVIEW

**Comment by gabecase** [comment]
@lekhajee this is for creating a server with a key, this is just the simple attribute change to the server object we discussed awhile back.

Actual key pair api endpoints are addressed here: https://github.com/rackerlabs/mimic/pull/436

File: mimic/test/test_nova.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def create_server(helper, name=None, imageRef=None, flavorRef=None,
     if body is None:
         data = {
             "name": name if name is not None else 'test_server',
+            "key_name": key_name if key_name is not None else 'test_key',
```
Comment (lekhajee) [inline mimic/test/test_nova.py]: Can we add an assert in the tests to ensure that the key_name when passed is set and when not passed is set to 'test_key'

**Comment by lekhajee** [comment]
Feel free to merge this once the tests are added and are passing travis. Thanks!

**Comment by lekhajee** [comment]
@gabecase : bump

**Comment by techvet** [comment]
thanks for the bump, was looking at the other PR. 

Will make your mentioned updates on this branch and merge by end of day

**Comment by gabecase** [comment]
thanks for the bump, was looking at the other PR. 

Will make your mentioned updates on this branch and merge by end of day

File: mimic/test/test_nova.py (hunk 1 above)
Comment (gabecase) [inline mimic/test/test_nova.py]: the two mentioned test cases were added below, plan to merge if the build succeeds.

**Pull request merged**